src/utils/dist/api.dev.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.getImage = void 0;
+
+var _axios = _interopRequireDefault(require("axios"));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
+
+var getImage = function getImage(setImageData) {
+  var _ref, _ref$data, urls, location, discrptions;
```

Comment:
same typo here

Reply:
this file is not valid , forgot to delete